docs/RELEASE_BRANCHES.md

```
@@ -21,43 +25,90 @@ git checkout -b release/YYYY-MM-DD
+
+```bash
+oc get pods -n rhel-lightspeed-stage --sort-by=.metadata.creationTimestamp
+```
+
+### 7. Production
+
+Only `rlsapi` has a production target in app-interface (SHA-pinned, 10 replicas). `lightspeed-stack`, `redhat-okp`, and `okp-mcp` are staging-only as of 2026-04-02.
 
 ## Deployment Flow
 
 ```
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)